### PR TITLE
Increase minimum julia version to v1.10

### PR DESCRIPTION
**Pull request by bennibolm**
[PR 2214](https://github.com/trixi-framework/Trixi.jl/pull/2214) increase the minimum required Julia version for Trixi.jl to v1.10. This PR updates it for Trixi2Vtk.jl.

### CODE REVIEW

File: test/test_trixi2vtk.jl [hunk 1]
```diff
@@ -53,11 +53,7 @@ end
 # Note: The purpose of using a specific commit hash (instead of `main`) is to be able to tie a given
 #       version of Trixi2Vtk to a specific version of the test file repository. This way, also tests
 #       for older Trixi2Vtk releases should continue to work.
-if VERSION < v"1.8"
-  const TEST_REFERENCE_COMMIT = "c0a966b06489f9b2ee3aefeb0a5c0dae733df36f"
-else
-  const TEST_REFERENCE_COMMIT = "e51f3613ac1adfcfd2bf1d74a2756034dab0579c"
-end
+const TEST_REFERENCE_COMMIT = "e51f3613ac1adfcfd2bf1d74a2756034dab0579c"
```
Comment (bennibolm) [inline test/test_trixi2vtk.jl:56]: Does it still make sense to keep testing with a specific commit hash instead of `main`?

Comment (sloede) [reply test/test_trixi2vtk.jl:56]: Yes, I think the reasoning in the comment still holds, don't you?

**Comment by sloede** [comment]
For some reason, the p4est tests fail. Do they just need updated reference files or what's the deal here?

**Comment by bennibolm** [comment]
> For some reason, the p4est tests fail. Do they just need updated reference files or what's the deal here?

I was wondering about that as well.
First, I wasn't even sure, if it would work to just set the commit hash to the commit in my branch as I did in the last commit. Still not fully sure, if it should work :sweat_smile: So, I still have some hope that the errors disappear when the reference files are updated.
Secondly, @benegee also got similar result in the [T8codeMesh-Trixi2Vtk PR](https://github.com/trixi-framework/Trixi2Vtk.jl/pull/95). His Windows tests also work, but Ubuntu and macOS don't. There, we were very confused that some tests even work at all, since there are no T8codeMesh files in the reference file repository yet.
It seems to have something to do with the function [`get_reference_file`](https://github.com/trixi-framework/Trixi2Vtk.jl/blob/58b6870d2e11cea16f8458d422d78abdbe23b574/test/test_trixi2vtk.jl#L29-L50) and that it just uses some sort of local files when the file path doesn't exists(?). Please correct me if I got that wrong.

**Comment by sloede** [comment]
> It seems to have something to do with the function [`get_reference_file`](https://github.com/trixi-framework/Trixi2Vtk.jl/blob/58b6870d2e11cea16f8458d422d78abdbe23b574/test/test_trixi2vtk.jl#L29-L50) and that it just uses some sort of local files when the file path doesn't exists(?).

By default it uses a file it finds locally, such that multiple tests using the same input file do not have to download it over and over again. A quick test if the old files somehow "pollute" the tests is by setting the default value of the `force` kwarg to `true`, such that each reference file must be downloaded anew each time it is used. If this fixes the issues, it would be a strong indicator that some caching is going wrong and we can fix it.

**Review by sloede — APPROVED** [review]
LGTM, thanks!

**Pull request merged**